src/sage/stats/distributions/discrete_gaussian_lattice.py

```diff
@@ -3,28 +3,32 @@
 Discrete Gaussian Samplers over Lattices
 
 This file implements oracles which return samples from a lattice following a
-discrete Gaussian distribution. That is, if `σ` is big enough relative to the
-provided basis, then vectors are returned with a probability proportional to
-`\exp(-|x-c|_2^2/(2σ^2))`. More precisely lattice vectors in `x ∈ Λ` are
-returned with probability:
+discrete Gaussian distribution. That is, if `\sigma` is big enough relative to
+the provided basis, then vectors are returned with a probability proportional
+to `\exp(-|x - c|_2^2 / (2\sigma^2))`. More precisely lattice vectors in `x \in
+\Lambda` are returned with probability:
 
-    `\exp(-|x-c|_2^2/(2σ²))/(∑_{x ∈ Λ} \exp(-|x|_2^2/(2σ²)))`
+.. MATH::
+
+    `\frac{\exp(-|x - c|_2^2 / (2\sigma^2))}{\sum_{v \in \Lambda} \exp(-|v|_2^2 /
+    (2\sigma^2))}`
```

Comment:
```suggestion
    \frac{\exp(-|x - c|_2^2 / (2\sigma^2))}{\sum_{v \in \Lambda} \exp(-|v|_2^2 /
    (2\sigma^2))}.
```
The extra backticks are not needed here.